src/components/List/List.tsx

```diff
@@ -87,8 +89,21 @@ export const List = <Item extends Entity>(props: ListProps<Item>) => {
                   : renderSkeleton}
               </Grid>
             ))
-          : data.items.map((item) => (
-              <Grid {...ItemProps} {...DataItemProps} item key={item.id}>
+          : data.items.map((item, index) => (
+              <Grid
+                {...ItemProps}
+                {...DataItemProps}
+                item
+                key={item.id}
+                sx={
+                  wideList
```

Comment:
I added this input so that we do not have to create an entirely new list component for this ticket. It made sense to me since it is just a smallish UI change but let me know if you think there is a better way